_includes/article.html

```diff
@@ -42,8 +42,7 @@ <h2 class="entry-title">{% if site.titlecase %}{{ page.title | titlecase }}{% el
         Categories: {% include post/categories.html %}
       </p>
       {% if page.series %}
-       <p class="meta-series">Part of a series: <strong>{{page.series}}</strong></p>
-       <h1>{{site.series_posts}}</h1>
+       <p class="meta-series" style="padding-bottom:0;">Part of a series: <strong>{{page.series}}</strong></p>
```

Comment:
I'd like to, when I took a stab at it, I couldn't get it working, I'll take another shot next time I look at this